lib/minify.js

```diff
@@ -0,0 +1,168 @@
+exports.readFile = function() {
+    DefaultsError.croak("readFile not supported");
+};
+
+exports.writeFile = function() {
+    DefaultsError.croak("writeFile not supported");
+};
+
+exports.simple_glob = function(files) {
+    return files;
```

Comment:
concat handles this flexible, non-arrays are implicitly treated as an array with 1 element.

https://github.com/mishoo/UglifyJS2/pull/1366/files/0750f4a338cfdc557d96fc68382751c0b23d873a#diff-29c0119b6a96bae34223c40f1c7f774bL79